drupal/sites/all/modules/antistatique_article/src/blocks/warning.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function antistatique_article_warning_block($param = null) {
+    $variables = array();
+    global $language;
+
+    $node = menu_get_object();
+    if ($node = menu_get_object() && $node->language != $language->language) {
```

Comment:
- `menu_get_object()` is called twice.
- what we want is to display the block only if the current language is `fr` and that there is no `en` translation. I don't think the condition achieve this goal.